### PR TITLE
Updating CLI instructions per DCOS-8814

### PR DESCRIPTION
**Pull request by mlunoe**

### CODE REVIEW

**Comment by tamarrow** [comment]
🚢

File: src/js/components/modals/CliInstallModal.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ class CliInstallModal extends React.Component {
         <div className="flush-top snippet-wrapper">
           <ClickToSelect>
             <pre className="prettyprint flush-bottom">
-              {`curl ${cliInstallScriptUrl} > /usr/local/bin/dcos && \n chmod +x /usr/local/bin/dcos && \n dcos config set core.dcos_url ${clusterUrl} && \n dcos`}
+              {`curl ${cliInstallScriptUrl} -o dcos && \n sudo cp dcos /usr/local/bin && \n sudo chmod +x /usr/local/bin/dcos && \n dcos config set core.dcos_url ${clusterUrl} && \n dcos`}
```
Comment (orlandohohmeier) [inline src/js/components/modals/CliInstallModal.js]: Why are we copying the dcos cli to `/usr/local/bin` instead of moving it?

Comment (mlunoe) [inline src/js/components/modals/CliInstallModal.js]: @tamarrow @jsancio any particular reason for this? Or can we safely move it?

Comment (tamarrow) [inline src/js/components/modals/CliInstallModal.js]: lets change to `mv`

Comment (mlunoe) [inline src/js/components/modals/CliInstallModal.js]: done

**Comment by mlunoe** [comment]
Rebased, and all comments addressed.

**Comment by mlunoe** [comment]
ping @jfurrow @MatApple

**Pull request merged**